packages/components/src/components/Drawer/DrawerBody.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Box } from "../../primitives/Box";
+
+export interface DrawerBodyProps
+  extends Omit<React.HTMLAttributes<HTMLDivElement>, "color"> {
+  /** The contents of the drawer body. */
+  children: NonNullable<React.ReactNode>;
+  /** Sets the padding of the drawer body. */
+  padding?: "space0" | "space60";
+}
+
+/** The body content area of the drawer. */
+const DrawerBody = React.forwardRef<HTMLDivElement, DrawerBodyProps>(
+  ({ children, padding = "space60", ...props }, ref) => {
+    return (
+      <Box.div
+        overflowY="auto"
+        padding={padding}
+        ref={ref}
+        {...props}
+        background="
+        linear-gradient(
+          white 30%,
+          rgba(255, 255, 255, 0)
+        ) center top,
+        radial-gradient(
+          farthest-side at 50% 0,
+          rgba(0, 0, 0, 0.2),
+          rgba(0, 0, 0, 0)
+        ) center top"
```

Comment:
Shouldn't this be a token of the DS? I mean, I can't see this on [Figma](https://www.figma.com/file/uDEjU77uIrQ5T1csn8IvIJ/Pluto-Tokens?node-id=1%3A18&t=uoBTEB0BP2UvHwMs-1), for example.

Reply:
Possibly. Right now its a one off use case to give show a shadow on scrolled content. If we find that we're reusing it more, we can add it in the future.